cookbook/Chapter 3 - Which borough has the most noise complaints (or, more selecting data).py

```diff
@@ -1,93 +1,50 @@
-# %%
-import pandas as pd
+import polars as pl
 import matplotlib.pyplot as plt
 
-
-# Make the graphs a bit prettier, and bigger
-plt.style.use("ggplot")
-plt.rcParams["figure.figsize"] = (15, 5)
-
-# This is necessary to show lots of columns in pandas 0.12.
-# Not necessary in pandas 0.13.
+# To display more columns in Polars
 pd.set_option("display.width", 5000)
```

Comment:
Looks good mostly! Just the pd references here throwing it off slightly.

Reply:
It looks it is done by pandas. But the requirement is to do in Polars.

Reply:
Thanks! I have made the changes